packages/generator-helper/src/__tests__/generatorHandler.test.ts

```diff
@@ -73,12 +75,15 @@ describe('generatorHandler', () => {
     }
   })
 
-  // TODO: Windows: this test fails with ENOENT even though the .cmd file is there and can be run manually.
+  // TODO: Windows: this test fails with ENOENT on CI because it can't file the .cmd file:
+  //   spawn D:\\a\\prisma\\prisma\\packages\\generator-helper\\src\\__tests__\\invalid-executable.cmd ENOENT
+  // This does not happen on Windows locally.
```

Comment:
Nothing changed here, just updated the comment to be more detailed. The the previous review comment for more info.